composer.json

```diff
@@ -77,5 +77,59 @@
             "ergebnis/composer-normalize": true
         },
         "sort-packages": true
+    },
+    "scripts": {
+        "cs:check": "php php-cs-fixer fix --dry-run",
+        "cs:diff": "git diff HEAD --name-only --diff-filter=ACMR -- '*.php' | xargs -r php php-cs-fixer fix --dry-run --diff",
+        "cs:diff-fix": "git diff HEAD --name-only --diff-filter=ACMR -- '*.php' | xargs -r php php-cs-fixer fix",
```

Comment:
kind-of :-1: here

imagine you work on your own project and we don't have cache feature. this makes sense.
but we have cache feature, we shouldn't promote running the fixer on modified files only, because we promote to have whole project fixed and following the rules. and if you change one file, the cache mechanism will make sure only one file got changed.

then, you work on Fixer project. you changed the `XyzFixer` to fix more cases. you don't want to run `fix` command on that one file only. you nee to run it on whole codebase, as you changed the fixer definition.

Reply:
FYI: cache doesn't work for me locally. I have `.php-cs-fixer.php` created from dist file, but 2 consequent runs take the same time (~47s):

<img width="862" alt="image" src="https://user-images.githubusercontent.com/600668/225057423-2ad0c5c6-a4a4-4c5f-a60c-994b02bdea60.png">

Shouldn't second run be much quicker? Even if I do:

```php
$config
    ->setUsingCache(true)
    ->setCacheFile(__DIR__ . '/.php-cs-fixer.cache')
```

the cache file is not created regardless of `--dry-run` (either when I only check or fix). Weird, but it's out of scope of this PR, so we can discuss it later.